airflow-core/src/airflow/ui/src/layouts/Nav/PluginMenus.tsx

```diff
@@ -28,6 +28,9 @@ import { Menu } from "src/components/ui";
 import { NavButton } from "./NavButton";
 import { PluginMenuItem } from "./PluginMenuItem";
 
+// Define existing button categories to filter out
+const existingCategories = ["user", "docs", "admin", "browse"];
```

Comment:
Are existing (v2 Plugins) using lower case categories? I would have assumed they use the non translated strings like "Docs" - do we need to match potentially on the legacy as well?